administrator/language/en-GB/en-GB.com_joomlaupdate.ini

```
@@ -58,8 +59,8 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE="Upd
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Please update these extensions before updating Joomla.</p><p id='updateorangewarning' class='hidden'>Please take extra care if this updated version of the extension is not also listed as compatible with your current version of Joomla.</p>"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_UPDATE_SERVER_OFFERS_NO_COMPATIBLE_VERSION="Update Information Unavailable"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_UPDATE_SERVER_OFFERS_NO_COMPATIBLE_VERSION_NOTES="Extension does not use the Joomla update system or the developer has provided no compatibility information."
-COM_JOOMLAUPDATE_VIEW_DEFAULT_SHOW_LESS_EXTENSION_COMPATIBILITY_INFORMATION="[ - ]"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_SHOW_MORE_EXTENSION_COMPATIBILITY_INFORMATION="[ + ]"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_SHOW_LESS_EXTENSION_COMPATIBILITY_INFORMATION="[ Less Detail %s ]"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_SHOW_MORE_EXTENSION_COMPATIBILITY_INFORMATION="[ More Detail %s ]"
```

```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_SHOW_LESS_EXTENSION_COMPATIBILITY_INFORMATION="[ Less Details %s ]"
COM_JOOMLAUPDATE_VIEW_DEFAULT_SHOW_MORE_EXTENSION_COMPATIBILITY_INFORMATION="[ More Details %s ]"
```
Always plural when using more or less for countable things.

---

@GeraintEdwards In case if I'm wrong with my above suggestion (what could be), just mark the conversation as resolved.

---

In english I would argue we can use 'More Detail' and 'Less Detail' here.  If you were talking to a student about her poor  exam result and she asked "what was wrong with my answers?" you could say "I needed more detail".

If we switch to the plural we we should use 'Fewer Details' not 'Less Details'

Don't know about other languages?  

Happy to go with singular or plural?

"More Information"/"Less Information" may be an alternaitve

---

Yes, with "information" only singular is possible. And you are right, with countable stuff it's "fewer" and not "less". I'd suggest we ask @brianteeman to check the 2 language strings and suggest corrections if necessary. Personally I like the "More Information"/"Less Information" alternative.

---

I agree with @GeraintEdwards

---

An interesting discussion here  on the topic https://forum.wordreference.com/threads/detail-vs-details.2822385/

---

I see I'm never too old to learn something new 😄 . Thanks. You can mark it as resolved if you want.